### PR TITLE
Replacing git protocol.

### PR DESCRIPTION
**Pull request by Karthick-Somasundaresan**
Github will stop supporting git protocol.
https://github.blog/2021-09-01-improving-git-protocol-security-github/#no-more-unauthenticated-git

Changing git:// protocol to https://

### CODE REVIEW

**Review by HaseenaSainul — APPROVED** [review]
LGTM

**Comment by krp97** [comment]
@HaseenaSainul is this PR going to be merged? I'm hitting on this now and there's a lot of failures related to the fact that we're using the git protocol. 

@Karthick-Somasundaresan I think you might've missed the `wpeframework.inc` file, because I'm getting an error for that recipe as well.

**Comment by HaseenaSainul** [comment]
@Karthick-Somasundaresan Could you please resolve the conflict ?

File: recipes-wpe/wpeframework/wpeframework-clientlibraries_git.bb [hunk 1]
```diff
@@ -12,9 +12,11 @@ require include/compositor.inc
 PR = "r0"
 PV = "3.0+gitr${SRCPV}"
 RECIPE_BRANCH ?= "master"
-SRC_URI = "git://github.com/rdkcentral/ThunderClientLibraries.git;protocol=git;branch=${RECIPE_BRANCH}"
+
+SRC_URI = "git://github.com/rdkcentral/ThunderClientLibraries.git;protocol=git;branch=${RECIPE_BRANCH};protocol=https"
 SRCREV ?= "504fb4cde12d05d5342b0c74279ccbdb455cc4e6"
 
+
```
Comment (HaseenaSainul) [inline recipes-wpe/wpeframework/wpeframework-clientlibraries_git.bb:19]: Please remove this additional lines

File: recipes-wpe/wpeframework/wpeframework-interfaces_git.bb [hunk 2]
```diff
@@ -11,9 +11,11 @@ DEPENDS_append = " wpeframework-tools-native wpeframework"
 PR = "r0"
 PV = "3.0+gitr${SRCPV}"
 RECIPE_BRANCH ?= "master"
-SRC_URI = "git://github.com/rdkcentral/ThunderInterfaces.git;protocol=git;branch=${RECIPE_BRANCH}"
+
+SRC_URI = "git://github.com/rdkcentral/ThunderInterfaces.git;protocol=git;branch=${RECIPE_BRANCH};protocol=https"
 SRCREV ?= "bf5de3c2cf64b5abc6e04928b2480349972c409b"
 
+
```
Comment (HaseenaSainul) [inline recipes-wpe/wpeframework/wpeframework-interfaces_git.bb:18]: Remove additional lines here as well

**Comment by HaseenaSainul** [comment]
Please replace protocol here as well: https://github.com/WebPlatformForEmbedded/meta-wpe/blob/development/replace_gitProtocol/recipes-wpe/wpeframework/include/wpeframework.inc#L10

**Review by HaseenaSainul — APPROVED** [review]
LGTM

**Review by krp97 — APPROVED** [review]
@Karthick-Somasundaresan's changes are working for me now. I think that we should go ahead without Artur's review, because he's on sick leave right now.

**Pull request merged**